### PR TITLE
Feature/pc 12 damage types

### PR DESCRIPTION
**Pull request by CuriousCactus**
Adds damage type categories and first level dice rolls for spells.
Also adds small symbols in the list of spells.

(first draft)

### CODE REVIEW

File: src/data/spells.json [hunk 1]
```diff
@@ -14,7 +14,9 @@
     "s_school": "Conjuration",
     "s_ritual": false,
     "s_name": "Acid Splash",
-    "s_description": "You hurl a bubble of acid. Choose one creature within range, or choose two creatures within range that are within 5 feet of each other. A target must succeed on a Dexterity saving throw or lake 1d6 acid damage. This spell's damage increases by 1d6 when you reach 5th level (2d6), 11th leveI (3d6), and 17th level (4d6)."
+    "s_damage_type": "acid",
+		"s_damage_dice": "1d6",
+		"s_description": "You hurl a bubble of acid. Choose one creature within range, or choose two creatures within range that are within 5 feet of each other. A target must succeed on a Dexterity saving throw or lake 1d6 acid damage. This spell's damage increases by 1d6 when you reach 5th level (2d6), 11th leveI (3d6), and 17th level (4d6)."
```
Comment (prophet-kn) [inline src/data/spells.json]: the spacing is odd here /o\

**Comment by prophet-kn** [comment]
Umh....

You really should not merge your own feature branch into a feature branch, lol

**Comment by prophet-kn** [comment]
Also the json still has weird spacing issues. :(

File: src/components/items/svgs/acid.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" style="height: 512px; width: 512px;"><path d="M0 0h512v512H0z" fill="#000" fill-opacity="1"></path><g class="" transform="translate(0,0)" style="touch-action: none;"><path d="M256.875 16A30 30 0 0 0 226 46a30 30 0 0 0 60 0 30 30 0 0 0-29.125-30zm-45 75A30 30 0 0 0 181 121a30 30 0 0 0 60 0 30 30 0 0 0-29.125-30zm74.563 30A15 15 0 0 0 271 136a15 15 0 0 0 30 0 15 15 0 0 0-14.563-15zm-30 45A15 15 0 0 0 241 181a15 15 0 0 0 30 0 15 15 0 0 0-14.563-15zM196 196c-45 0-15 30 0 45 0 150-120 225-120 255h360c0-30-120-105-120-255 15-15 45-45 0-45-15 0-30 15-60 15s-45-15-60-15z" fill="#fff" fill-opacity="1"></path></g></svg>
```
Comment (prophet-kn) [inline src/components/items/svgs/acid.svg]: Can you remove the xmlns values as well as any colours, such as fill and others that might exist, in all of the SVG files?

File: src/components/toggle-button/toggle-button.js [hunk 3]
```diff
@@ -29,8 +29,13 @@ class ToggleButton extends React.Component {
       classNames.push('selected')
     }
 
+    var label = ""
```
Comment (prophet-kn) [inline src/components/toggle-button/toggle-button.js]: We don't need label to be defined on function level, just block, can you make it 'let' and not 'var'?

File: src/styles/_main.scss [hunk 4]
```diff
@@ -546,6 +546,14 @@
     z-index: 1;
   }
 
+  &-damage-icon {
+    width: 24px;
+    cursor: pointer;
+    right: 122px;
```
Comment (prophet-kn) [inline src/styles/_main.scss]: Interesting, why?

File: src/styles/_main.scss [hunk 5]
```diff
@@ -546,6 +546,14 @@
...
+    position: absolute;
+    top: 15px;
+  }
+
```
Comment (prophet-kn) [inline src/styles/_main.scss]: Can you keep related attributes closer? Attributes that refer to position, such as 'right' etc, for code clarity should be defined after setting position. Sure you might say it's all compiled in the end, but we don't work on the compiled files and makes it look messy. :)

**Comment by CuriousCactus** [comment]
I think this was actually nearly done but just didn't have time to check it before I last pushed. Now ready for review.

**Comment by prophet-kn** [comment]
Due to a lot of stuff being merged, can you fix conflicts?

**Comment by CuriousCactus** [comment]
I think this is ready!

**Comment by CuriousCactus** [comment]
Actually done now, just tweaked when the damage line shows.

**Comment by prophet-kn** [comment]
Yup, the last change you did was needed. :) I will adjust Prismatic Wall and Prismatic Spray as their damage is based on the die, from 'cold' to 'various'. Otherwise, nice! Merging.

**Pull request merged**